bin/check.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
```

Comment:
you seem to have lost history on this file (it was added/and the original removed below, rather than moved). However, I don't mind/care, and it's a pain to sort out. Just an fyi for the future.